index.html

```diff
@@ -349,6 +349,23 @@ <h2>About Myself</h2>
 
 
 
+      </section>
+      <section class="contact-form">
+        <h2>Contact Me</h2>
+        <p>If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding.
+          I’d love to help with it!</p>
+        <form action="/contact" method="post" enctype="multipart/form-data">
+          <input type="text" placeholder="Full names"
+            max="25" min="10"
+            required="required">
+          <input type="text" placeholder="email" min="10"
+            required="required">
+          <textarea rows="7" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] Kindly ensure that your Message(textarea) input has `maxlength="500"` attribute.